sharktank_models/benchmarks/run_benchmarks.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2025 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import subprocess
+import os
+from pathlib import Path
+import argparse
+import sys
+import tabulate
+import json
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--model", type=str, default="sdxl")
+    parser.add_argument("--filename", type=str, default="*")
+    parser.add_argument("--sku", type=str, default="mi300")
+    parser.add_argument("--backend", type=str, default="gfx942")
+    args = parser.parse_args()
+    model = args.model
+    filename = args.filename
+    sku = args.sku
+    backend = args.backend
+
+    os.environ["BENCHMARK_MODEL"] = model
+    os.environ["BENCHMARK_FILE_NAME"] = filename
+    os.environ["SKU"] = sku
+    os.environ["BACKEND"] = backend
+
+    THIS_DIR = Path(__file__).parent
+
+    command = [
+        "pytest",
+        THIS_DIR / "test_model_benchmark.py",
+        "--log-cli-level=info",
+        "--timeout=600",
+        "--retries=0",
+    ]
+    
+    with open("job_summary.md", "a") as job_summary, open("job_summary.json", "w+") as content:
+        print(f"{sku.upper()} {model.upper()} Complete Benchmark Summary:\n", file=job_summary)
+        json.dump({}, content)
```

Comment:
May want to add this file to `.gitignore` if it isn't already so if a developer runs this locally it doesn't pollute their git index.

Another thing you can do is detect if you are running in a github action and change the behavior based on that. Plenty of ways to do that. Stella added one recently in https://github.com/nod-ai/TheRock/blob/main/build_tools/teatime.py with an environment variable that the script checks for.